### PR TITLE
Clean up entity model, remove extra states and attributes

### PR DESCRIPTION
**Pull request by iMicknl**
- Remove extra states and attributes
- Improve device info typing
- Use extra_state_attributes (new in core)
- Only use the actual firmware for firmware

### CODE REVIEW

File: custom_components/tahoma/entity.py [hunk 1]
```diff
@@ -88,40 +89,28 @@ def device_info(self) -> dict[str, Any]:
             or self.device.widget
         )
 
-        return {
-            "identifiers": {(DOMAIN, self.executor.base_device_url)},
-            "manufacturer": manufacturer,
-            "name": self.device.label,
-            "model": model,
-            "sw_version": self.device.controllable_name,
```
Comment (tetienne) [inline custom_components/tahoma/entity.py:96]: I would like to see the device.controllable_name somewhere. It helps me do retrieve the setup.

Comment (iMicknl) [reply custom_components/tahoma/entity.py:96]: Same! I would propose to add this to system_health and update our README? Or only keep this in the debug logging and ask people to add their debug logs.

It doesn't make a lot of sense to have this visible in the main interface, if we only need it for debug purposes. Agree?

Comment (tetienne) [reply custom_components/tahoma/entity.py:96]: Agree! Can you add the debug log with this PR?

Comment (iMicknl) [reply custom_components/tahoma/entity.py:96]: @tetienne don't we already have all this information in the debug log? Only system health isn't in this PR yet.

**Review by tetienne — APPROVED** [review]

**Review by tetienne — APPROVED** [review]

**Comment by tetienne** [comment]
Can we merge this one?

**Pull request merged**